src/DocParser.php

```diff
@@ -98,7 +98,7 @@ private function incorporateLine(ContextInterface $context)
         $context->setContainer($context->getDocument());
         while ($context->getContainer()->hasChildren()) {
             $lastChild = $context->getContainer()->getLastChild();
-            if (!$lastChild->isOpen()) {
+            if (!is_null($lastChild) && !$lastChild->isOpen()) {
```

Comment:
I don't think this check is necessary.  Line 99 checks whether any children exist, so line 100 will always return a child object.